README.md

```diff
@@ -55,184 +22,76 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
+# 📖 [Capstone_project_1] <a name="about-project"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+> It is a project build with HTML,CSS and Javascript according to linters standards. Mobile responsive website version.
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+**[Capstone_project_1]** is a site practice of my own from Microverse
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+## Video Presentation Check this <a href="https://www.loom.com/share/1a4ad6ec101d49e5969dfc822fd59e87">link</a>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
+## 🛠 Built With HTML,CSS and Javascript <a name="built-with"></a>
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Build with semantic HTML tags]**
+- **[CSS responsive media queries added]**
+- **[CSS flexbox module used in this project]**
+- **[.gitignore has been set up properly]**
+- **[JS functionality added properly]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- https://github.com/Rifat8080/mahadi_capstone_project_1
```

Comment:
Please as suggested by the previous reviewer, you should add the deployed live link of your project here and not your github project link, so as to allow visitors have a glance at your work 😄